www/~/%username/emails/modify.json.spt

```diff
@@ -27,11 +27,18 @@ if not participant.email_lang:
 
 msg = None
 if action in ('add-email', 'resend'):
-    r = participant.add_email(address, website.env.resend_verification_threshold)
-    if r:
-        msg = _("A verification email has been sent to {0}.", address)
+    try:
+        participant.add_email(address, website.env.resend_verification_threshold)
+    except EmailTaken:
```

Comment:
Apparently we're not testing the exceptions here, because I just discovered down below #4348 that `EmailTaken` is never imported here.